docs/TRANSLATING.md

```diff
@@ -246,6 +262,39 @@ git push --force # Push the changes to your repo. Be aware that this will overwr
 
 ---
 
+## Adding a new language (checklist)
+
+The following section describes how to add a new language and what to do or wait for.
+Please open an issue on GitHub and introduce yourself first, even if you intend to add the translation files yourself.
+
+After having opened an issue on GitHub:
+1. Fork the Jamulus repository as described above, then install Qt on your maching. (You will need `lrelease` and `lupdate` later on.)
+2. Create a branch based on the `master` branch: Locally navigate to the folder you cloned the repo with `cd /path/to/jamulus/repo` and type `git checkout -b "translate-add-xx_XX"` where xx_XX is the language code of the new language.
+3. Open the Jamulus.pro file in an editor such as `nano`: `nano Jamulus.pro` and add your language file (which will be created later) in alphabetical order to the `TRANSLATIONS` variable. The format should match the other present languages.
+4. Also add `src/translation/translation_xx_YY.qm` to the DISTFILES variable similar to the other files
+5. Save and close Jamulus.pro.
+6. Add the `.qm` file in `src/resources.qrc` similar to the other languages.
+7. Now move back to the repository root and run `lupdate Jamulus.pro` and afterwards `lrelease Jamulus.pro` which should create all of the necessary files.
+8. Finally, add, commit and push your changes to your repository with the following commands: `git add src/translation/translation_xx_YY.* src/resources.qrc`, `git commit -m "Translation: Add empty xx_YY translation"`, `git push`
```

Comment:
your changes to your repository sounds strange.

Reply:
Just drop "to your repository".